src/VecSim/algorithms/brute_force/brute_force.cpp

```diff
@@ -1,7 +1,6 @@
 
 #include "brute_force.h"
-#include "VecSim/spaces/L2_space.h"
-#include "VecSim/spaces/IP_space.h"
+#include "VecSim/spaces/spaces.h"
```

Comment:
already included in h file

Reply:
it is guarded with pragma #once
isnt this considred as good practice that each file explictly includes all its dependencies?